class_generator/tests/test_class_generator.py

```diff
@@ -16,12 +16,13 @@
         "ConfigMap",
         "DNS",
         "Deployment",
+        "Image",
```

Comment:
No need to add a test for this resource.

Reply:
Done